ovirt/provider.go

```diff
@@ -79,7 +79,7 @@ var providerSchema = map[string]*schema.Schema{
 
 // New returns a new Terraform provider schema for oVirt.
 func New() func() *schema.Provider {
-	return newProvider(ovirtclientlog.NewNOOPLogger()).getProvider
+	return newProvider(newTerraformLogger()).getProvider
 }
 
 func newProvider(logger ovirtclientlog.Logger) providerInterface {
```

Comment:
The call to `ovirtclient.NewTestHelper` could be simplified with the new ovirtclient version to 
```go
helper, err := ovirtclient.NewMockTestHelper(logger)
```